test/es-module/test-esm-export-not-found.mjs

```diff
@@ -0,0 +1,37 @@
+import '../common/index.mjs';
+import { path } from '../common/fixtures.mjs';
+import { match } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+const importStatement =
+  'import { foo, notfound } from \'./module-named-exports.mjs\';';
+const importStatementMultiline = `import {
+  foo,
+  notfound
+} from './module-named-exports.mjs';
+`;
+
+[importStatement, importStatementMultiline].forEach((input) => {
+  const child = spawn(execPath, [
+    '--input-type=module',
+    '--eval',
+    input,
+  ], {
+    cwd: path('es-module-loaders'),
+  });
+
+  let stderr = '';
+  child.stderr.setEncoding('utf8');
+  child.stderr.on('data', (data) => {
+    stderr += data;
+  });
+  child.on('close', () => {
```

Comment:
I'd add a `mustCall` here and in the other files. And maybe always assert the exit code (it's passed to the callback)?

Reply:
Please add the `mustCall`, it's cheap and makes the tests more reliable. If the callback never runs, the test may or may not time out, or it could finish silently.

For the exit code, I think `assert.strictNotEqual(code, 0)` would be enough.